### PR TITLE
added language contributor data to cldf #5; added form 666 (coming fr…

### PR DESCRIPTION
**Pull request by Bibiko**
…om Peiros data) as being invalid #3; added source key1984 (was hidden as consultant)

### CODE REVIEW

**Review by xrotwang — COMMENTED** [review]
Merging this would mean we cannot regenerate `raw` by running `download` anymore, correct? If so, `cmd_download` should be removed.

**Comment by Bibiko** [comment]
yes - I'll remove it

**Comment by Bibiko** [comment]
there was no cmd_download since the app's initDB made usage of the original data

File: cldf/forms.csv [hunk 1]
```diff
@@ -213338,7 +213338,7 @@ ID,Local_ID,Language_ID,Parameter_ID,Value,Form,Segments,Comment,Source,Cognacy,
 243-11-430-1,,243,11-430,nyia,nyia,,"Also means 'gold, silver' 09.640, 09.650.",perez1987,,,latintrans,nɨia,phonemic
 243-11-510-1,,243,11-510,zipquazagasqua,zipquazagasqua,,"Means 'to make rich, become rich'.",perez1987,,,latintrans,c̷̣-ipkua-c̷̣-aɣa-skua,phonemic
 243-11-540-1,,243,11-540,ataban mague,ataban mague,,,perez1987,,,latintrans,a-taba-n maɣe,phonemic
-243-11-620-1,,243,11-620,--btyusuca,--btyusuca,,Object precedes verb.,perez1987,,,latintrans,--β-tɨu-suka,phonemic
+243-11-620-1,,243,11-620,-btyusuca,-btyusuca,,Object precedes verb.,perez1987,,,latintrans,-β-tɨu-suka,phonemic
```
Comment (xrotwang) [inline cldf/forms.csv]: What is this normalization based on? Above, a leading `-`is removed, here it's left in?

Comment (Bibiko) [reply cldf/forms.csv]: at Key's original data CD-ROM a em-dash was loaded as '--' which was nothing else as a normal hyphen

**Review by xrotwang — CHANGES_REQUESTED** [review]
I'm not really sure what the changes of the raw forms are based on. Did we get new data from somewhere?
If not, I'd argue that the changes should
- either go into processing code (i.e. `lexibank_ids.py`)
- or config data in `etc/lexemes.csv`

Otherwise we lose transparency as to what came from whom.

**Comment by xrotwang** [comment]
@Bibiko the `cmd_download` is inherited from `pylexibank.providers.clld.CLLD` - so must be overwritten to explicitly disable it.

**Comment by xrotwang** [comment]
But why is the single hyphen above removed?

Hans-Jörg Bibiko <notifications@github.com> schrieb am Mo., 21. Okt. 2019,
17:12:

**Comment by Bibiko** [comment]
the exported csv ids-data were only the 'half of the truth' - lots of stuff was normalized within the cgi-script

**Comment by xrotwang** [comment]
I see. But if this normalization happened in code before, it could happen
in code again, no?

Hans-Jörg Bibiko <notifications@github.com> schrieb am Mo., 21. Okt. 2019,
17:42:

**Comment by Bibiko** [comment]
no, it does not happen again

**Comment by Bibiko** [comment]
I'll move all the normalized data to etc/lexemes.csv for transparency - btw, where is the script which generated the first raw/forms.csv? In addition, the used ids-data csv files contain lots of old data from Mazhid's Caucasisan project/book (misaligned forms, Russian characters within Phonemic, different diacritical characters [which led to counterpart problems], some of the phonemic transliterations where done quickly after the publication, etc. etc.)

**Comment by xrotwang** [comment]
@Bibiko I don't really understand. It may not happen right now, but it could happen in `lexibank_ids.py`. That's what I meant. The idea of the input directories in lexibank is
- `raw`: whatever came from the original authors
- `etc` and `lexibank_*.py`: whatever we add to "enhance" the resulting
- data in `cldf`.

Following this logic, what used to be in the CGI should now go into either `etc` or `lexibank_ids.py`.

**Comment by xrotwang** [comment]
The data in `raw` right now comes from https://ids.clld.org/download which in turn comes from the csv tables you sent me years ago.

**Comment by Bibiko** [comment]
ok - as I said, I'll move all the normalized/enhanced data to etc/lexemes.csv for transparency - my main point here is to be able to come up with a CLDF dataset which can be read by the ids app's initDB (since up to now the current initDB make usage of the `ids-data` set which is not what one gets if you go via https://ids.clld.org/download.

**Comment by Bibiko** [comment]
And if remember correctly I sent you these data https://github.com/clld/ids-data - that's why I asked which script was used to come up with the data https://github.com/lexibank/ids.

**Comment by xrotwang** [comment]
yes, it's more complicated. `clld/ids-data` was the input for the clld app db. Then the app was used to create the CLDF dataset available from the download page. So yes, there's already some processing being done, and yes, it might be ideal to be able to trace this as well. But then, that happened already years ago, and that's basically the justification for the CLDF dataset in `raw` being regarded as the source.

**Comment by chrzyki** [comment]
Can this be closed in favour of a new PR @Bibiko if still needed?

**Pull request closed without merging**